md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTemplateAPI.java

```diff
@@ -374,18 +342,14 @@ RestWrapper create(@ModelAttribute("processtemplate")
                 if (index > 0) {
                     processTempInfo.setParentProcessId(pid);
                 }
-
-                // processes.add(index, (Process) s.selectOne("call_procedures.InsertProcess", processTempInfo));
                 pid = processes.get(0).getProcessId();
                 LOGGER.debug("index= " + index + "processTempInfo.processtempid=" + processTempInfo.getProcessTemplateId() + "processes.(0)name= " + processes.get(index).getProcessId());
 
                 // Inserting properties for newly created process from template
-                //     List<PropertiesTemplate> propertiesTemplateList = s.selectList("call_procedures.ListPropertiesOfProcessTemplate", processTempInfo);
                 List<PropertiesTemplate> propertiesTemplateList = propertiesTemplateDAO.listPropertiesTemplateBean(processTempInfo.getProcessTemplateId());
                 for (PropertiesTemplate propertiesTemplate : propertiesTemplateList) {
-                    if (propertiesTemplateList.size() != 0) {
+                    if (propertiesTemplateList.isEmpty()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)